<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAssayExportDlg</class>
 <widget class="BaseDlg" name="QAssayExportDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAssayExportDlg</string>
  </property>
  <widget class="QTableView" name="tableView_assay">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>1130</width>
     <height>629</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar::handle:vertical  {
    margin-top: 46px;
}</string>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ensure">
   <property name="geometry">
    <rect>
     <x>937</x>
     <y>806</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>1067</x>
     <y>806</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>1130</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>1</width>
     <height>629</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>778</y>
     <width>1130</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>1179</x>
     <y>150</y>
     <width>1</width>
     <height>629</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QWidget" name="" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>87</y>
     <width>1081</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>导出位置：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_path">
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_exPathSel">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>46</height>
       </size>
      </property>
      <property name="text">
       <string>选择</string>
      </property>
      <property name="buttons" stdset="0">
       <string notr="true">btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
